src/satosa/backends/saml2.py

```diff
@@ -84,12 +80,13 @@ class SAMLBackend(BackendModule, SAMLBaseModule):
     KEY_SP_CONFIG = 'sp_config'
     KEY_SEND_REQUESTER_ID = 'send_requester_id'
     KEY_MIRROR_FORCE_AUTHN = 'mirror_force_authn'
-    KEY_IS_PASSIVE = 'is_passive'
     KEY_MEMORIZE_IDP = 'memorize_idp'
     KEY_USE_MEMORIZED_IDP_WHEN_FORCE_AUTHN = 'use_memorized_idp_when_force_authn'
 
     VALUE_ACR_COMPARISON_DEFAULT = 'exact'
 
+    SAML_PARAMS = [Context.KEY_SAML_FORCE_AUTHN, Context.KEY_SAML_IS_PASSIVE]
```

Comment:
I don't think it is a good idea to mix the two params.

ForceAuthn is handle in one way.

IsPassive is introduced. It may affect parts of the handling of ForceAuthn but that shouldn't mean they should be bound together.

It is easier to review this if the new functionality is clearly separated from the old.

Reply:
@c00kiemon5ter They are handled similarly, so I did a little refactoring to prevent adding duplicate code. Do you want to rather split the code (which means duplicity)?